Tests/test_file_jpeg2k.py

```diff
@@ -174,15 +174,8 @@ def test_16bit_jp2_roundtrips(self):
     def test_unbound_local(self):
         # prepatch, a malformed jp2 file could cause an UnboundLocalError
         # exception.
-        try:
-            jp2 = Image.open('Tests/images/unbound_variable.jp2')
-            self.assertTrue(False, 'Expecting an exception')
-        except SyntaxError as err:
-            self.assertTrue(True, 'Expecting a syntax error')
-        except IOError as err:
-            self.assertTrue(True, 'Expecting an IO error')
```

Comment:
This one's a bit odd in that the original allows either a SyntaxError or IOError...